great_expectations/execution_engine/sqlalchemy_batch_data.py

```diff
@@ -217,11 +224,10 @@ def _create_temporary_table(
                 temp_table_name = f"{temp_table_schema_name}.{temp_table_name}"
 
             stmt = f"CREATE OR REPLACE TEMPORARY TABLE {temp_table_name} AS {query}"
-        elif dialect == GESqlDialect.MYSQL:
-            # Note: We can keep the "MySQL" clause separate for clarity, even though it is the same as the
-            # generic case.
+        elif dialect == GESqlDialect.MYSQL or dialect == GESqlDialect.HIVE:
             stmt = f"CREATE TEMPORARY TABLE {temp_table_name} AS {query}"
         elif dialect == GESqlDialect.MSSQL:
+
```

Comment:
Very minor: If we do not need this blank line, let us not add it.  Thank you.

Reply:
Removed 👍